src/util/string.h

```diff
@@ -650,7 +652,7 @@ std::vector<std::basic_string<T> > split(const std::basic_string<T> &s, T delim)
 	return tokens;
 }
 
-std::wstring translate_string(const std::wstring &s, const std::string &lang_code);
+std::wstring translate_string(const std::wstring &s, Translations *translations);
```

Comment:
does `const Translations*` not work here?
Const correctness might be simple to fix if it doesn't work yet.

Reply:
Seems not possible. If a missing translation is asked, an empty entry is added to translations list in order to avoid multiples warnings.

Reply:
Ugh. Can be left like this then.